feat(changelog): jobs-fixed-referencing-secret-with-custom-encryption-ke 2026-06-15 (#6603)
scaleway/docs-content: changelog/june2026/2026-06-15-jobs-fixed-referencing-secret-with-custom-encryption-ke.mdx

---

> title: Referencing Secret with custom Encryption Key

Reviewer: ```suggestion
title: Referencing a secret with a custom encryption key
```

status: fixed
date: 2026-06-15
category: serverless
product: jobs
---

> Referencing a secret from Secret Manager that uses a custom encryption key in a Job Definition would cause Job Runs to fail during the validation step. It is now fixed.

Reviewer: ```suggestion
Referencing a secret from Secret Manager that uses a custom encryption key in a job definition caused job runs to fail during the validation step. This issue is now fixed.
```
